src/views/auth/login.vue

```diff
@@ -1,66 +1,135 @@
 <style lang="scss" skoped>
   #wrapper .container-fluid {
-    background: black;
+    background: rgb(11,4,37);
+    background: radial-gradient(200% 150% ellipse at 50% 215%, rgba(190,100,154,1), rgba(11,4,37,1));
+    background-repeat: no-repeat;
+    background-attachment: fixed;
     min-height: calc(var(--vh, 1vh) * 100);
   }
 
   h1 { font-size: 22px; }
   form { min-width: 330px; }
+
+  #disclaimer{
```

Comment:
Dit is niet perce nodig, kan ook met klasses: 'small pt-5' en eventueel in de form steken zodat die de width daar van mee komt.